report/export.py

```diff
@@ -1 +1,24 @@
+import reports
 # Export functions
+
+def export(source_file_name,output_file_name):
```

Comment:
Erre van egy jobb módszer: definiálj docstring-et a függvényekhez, ami után a string-et el tudod érni így:
http://stackoverflow.com/questions/713138/getting-the-docstring-from-a-function

Ezzel két legyet üthetsz 1 csapásra: egyrészt kommenteled a függvényt, másrészt nem kell itt mindent újra leírnod.
Például eltárolhatnád a függvényeket egy listában, és az export csak annyit csinálna, hogy végigmegy a listán és kiírja a docsstring-et, aztán a függvény értékét.